Doc/tutorial/appendix.rst

```diff
@@ -31,6 +31,32 @@ Typing an interrupt while a command is executing raises the
 :exc:`KeyboardInterrupt` exception, which may be handled by a :keyword:`try`
 statement.
 
+.. _tut-var:
+
+Special ``_`` variable
+-------------------------
+
+There is a special variable ``_`` in interactive mode, stored in the :mod:`builtins` 
+module. It stores the result of the last evaluated expression (*except* ``None``
+— it remains unchanged).
```

Comment:
I think this would read better as: "... the result of the last evaluated expression. If the last expression evaluated to ``None``, then the value of ``_`` is not updated."

Reply:
Agree.